src/app.ts

```diff
@@ -24,4 +26,30 @@ void app.register(fastifyHealthcheck, {
   },
 })
 
+void app.register(fastifySwagger, {
+  mode: "dynamic",
+  openapi: {
+    openapi: "3.0.0",
+    info: {
+      title: "Transfers API",
+      version: "1.0.0",
+      description: "API documentation for Transfers API",
+    },
+    servers: [
+      {
+        url: "http://localhost:8000/",
```

Comment:
We probably want to make this configurable

Reply:
Or to be more clear this should a route on the existing API instance a not a new server